lib/index.js

```diff
@@ -330,6 +339,22 @@ Server.prototype = {
         eurekaOverrides: apimlConfig.eureka
       });
       yield this.apiml.registerMainServerInstance();
+      yield this.apiml.takeOutOfService();
+      const instanceId = this.apiml.getInstanceId();
+      let appServerClusterSize = +process.env.ZOWE_APP_SERVER_CLUSTER_SIZE;
+      if (!Number.isInteger(appServerClusterSize) || appServerClusterSize < 3) {
+        appServerClusterSize = 3;
+      }
```

Comment:
a code comment is needed in this section
1. why does it take out of service as soon as possible
2. why is the minimum cluster size 3?
I think the answer is in another file I haven't read yet, so a code comment will help people to understand.

Reply:
All instances must be no-op until Leader is elected. 3 is a minimal cluster size. With 3 peers we can tolerate 1 failure.